cmd/2019_8/main.go

```diff
@@ -0,0 +1,86 @@
+package main
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [the file doesn't have an appropriate header](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5573476)